### DISCUSSION TITLE
350p pip로 ebcli설치과정중 오류 발생합니다..

### DISCUSSION THREAD

**Discussion by ho9924** [fsoftwareengineer/todo-application#22] [질문과 답변] [answered]
윈도우 환경에서 ebcli 설치 후 eb --version명령어 입력 시 아래와 같이 뜹니다. 어떻게 해결해야 할까요?

SyntaxError: Non-UTF-8 code starting with '\xb1' in file C:\Users\한글이름\AppData\Roaming\Python\Python310\Scripts\eb-script.py on line 2, but no encoding declared; see https://python.org/dev/peps/pep-0263/ for details

**Comment by fsoftwareengineer** [comment, marked as answer]
파이썬 환경변수가 잘 설정되었는지 확인 해 주시기 바랍니다. (PATH와 PYTHONPATH)
파이썬고ㅓ Ebcli를 언인스톨 후 재인스톨 허는 방법도 있습니다. 

출처 https://stackoverflow.com/questions/68172380/awsebcli-syntaxerror-no-encoding-declared-in-terminal-upon-executing-any-eb-c
